sensu/plugins/check-fs-writable.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# checks that all mounts listed in fstab are writeable
+
+mount_status=0
+
+for mount in $( findmnt -sno target | grep -v none ); do
+  file="${mount}/.tmp-writable-check"
+  trap '[[ -f "${file}" ]] && rm "${file}"' TERM EXIT
+
+  if ! touch "$file" 2>/dev/null; then
+    echo "FAIL - Could not create a file into '${mount}' !"
```

Comment:
if the file already exists then `touch` won't be creating it.  Maybe swap `touch` for `create` in the FAIL message